build/storage/tests/it/ptf_tests/system_tools/ssh_terminal.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+from paramiko.client import AutoAddPolicy, SSHClient
+
+
+class CommandException(Exception):
+    """Occur command Error in terminal"""
+
+
+class SSHTerminal:
+    """A class used to represent a session with an SSH server"""
+
+    def __init__(self, config, *args, **kwargs):
+        self.config = config
+        self.client = SSHClient()
+
+        self.client.load_system_host_keys()
+        self.client.set_missing_host_key_policy(AutoAddPolicy)
+        self.client.connect(
+            config.ip_address,
+            config.port,
+            config.username,
+            config.password,
+            *args,
+            **kwargs
+        )
+
+    def execute(self, cmd: str, timeout: int = None) -> list:
+        """Simple function executes a command on the SSH server
+        Return list of the lines output
+        """
+        _, stdout, stderr = self.client.exec_command(cmd, timeout=timeout)
+        if stdout.channel.recv_exit_status():
+            raise CommandException(stderr.read().decode())
+        #  if command is executed in the background don't wait for the output
+        out = [] if cmd.rstrip().endswith("&") else stdout.readlines()
+        return [line.rstrip() for line in out]
+
+    # TODO: add tracking running containers while testing and kill only onest
+    def delete_all_containers(self):
```

Comment:
Are we sure we want to kill all containers in the system? Should we not focus on the containers relevant to storage setup/test cases?